snews.php

```diff
@@ -945,7 +945,7 @@ function articles() {
 			if ($result = db() -> query($query_articles)) {
 				$link = '<a href="'._SITE;
 				while ($r = dbfetch($result)) {
-					$infoline = $r['displayinfo'] == 'YES' ? true : false;
```

Comment:
Not sure which "best practice" you're referring to. Check http://c2.com/cgi/wiki?ReturnBooleanEvaluations